Obsidian/Assets/tags.json

```diff
@@ -98,22 +98,27 @@
       "minecraft:red_concrete_powder",
       "minecraft:black_concrete_powder",
       "minecraft:scaffolding",
-      "minecraft:dragon_egg"
+      "minecraft:dragon_egg",
+      "minecraft:gravel",
+      "minecraft:suspicious_gravel"
     ]
   },
   "blocks/replaceable_by_liquid": {
     "name": "replaceable_by_liquid",
     "type": "blocks",
     "values": [
-      "#minecraft:replaceable_by_trees",
```

Comment:
Why was this removed?

Reply:
because this contains leaves that should not be replaced by water